### ISSUE TITLE
loss = INF

### ISSUE THREAD

**Issue by MaximilianPi** [citoverse/cito#32]
<img width="513" alt="image" src="https://user-images.githubusercontent.com/29951520/214258951-d50bd912-1c2a-4631-96c5-a0b298d48988.png">
